crawler/main.js

```diff
@@ -75,7 +75,17 @@ configPromises.push(dataStore.configure({
 }));
 
 configPromises.push(urlJobProcessor.configure({
-
+  probes: [
+    {
+      name: 'totalRecords',
```

Comment:
Adds some noise to have a probe that doesn't actually probe. The measurement can be also done within the probe job.